integration-tests/tests/run-notebook.js

```diff
@@ -25,6 +25,8 @@ const testRunNotebookFn = _.flow(
   await select(page, 'Language', 'Python 3')
   await click(page, clickable({ text: 'Create Notebook' }))
   await click(page, clickable({ textContains: notebookName }))
+  //Adding a short microsleep to make sure that we finish waiting for the spinner to stop blocking the click
```

Comment:
My understanding of why we add the micro sleep was more like

```suggestion
  // We are adding a short micro sleep to make sure that the spinner is
  // rendered by the time we start waiting for the spinner to go away.
```

Am I missing something?

Cc: @breilly2

Reply:
Yes, that's exactly it. Thank you for boiling it down to the simplest explanation!